code/__DEFINES/~skyrat_defines/combat.dm

```diff
@@ -108,73 +108,84 @@
-	if(modifiers && modifiers[RIGHT_CLICK])
-		switch(user.zone_selected)
-			if(BODY_ZONE_HEAD)
-				if(target.body_position == LYING_DOWN)
-					. = TRUE
-					try_headslam(user, target, affecting)
-			if(BODY_ZONE_CHEST)
-				if(istype(user.mind.martial_art, /datum/martial_art/cqc)) //If you know CQC, You can't suplex and instead have the ability to use the chokehold, Sorry.
-					return
-				//Suplex!
-				. = TRUE
-				target.visible_message(span_danger("[user.name] holds [target.name] tight and starts lifting [target.p_them()] up!"), \
-						span_userdanger("[user.name] holds you tight and lifts you up!"), ignored_mobs=user)
-				to_chat(user, span_danger("You hold [target.name] tight and lift [target.p_them()] up..."))
-				user.changeNext_move(SUPLEX_TIMER)
-				if(do_after(user, SUPLEX_TIMER, target))
-					var/move_dir = get_dir(target, user)
-					var/moved_turf = get_turf(target)
-					for(var/i in 1 to 2)
-						var/turf/next_turf = get_step(moved_turf, move_dir)
-						if(IS_OPAQUE_TURF(next_turf))
-							break
-						moved_turf = next_turf
-					target.visible_message(span_danger("[user.name] suplexes [target.name] down to the ground!"), \
-						span_userdanger("[user.name] suplexes you!"), ignored_mobs=user)
-					to_chat(user, span_danger("You suplex [target.name]!"))
-					user.StaminaKnockdown(30, TRUE, TRUE)
-					user.SpinAnimation(7,1)
-					target.SpinAnimation(7,1)
-					target.throw_at(moved_turf, 1, 1)
-					target.StaminaKnockdown(80)
-					target.Paralyze(2 SECONDS)
-					// feels like there should be some funny brute here but iunno
-					log_combat(user, target, "suplexes", "down on the ground (knocking down both)")
-			else
-				var/datum/wound/blunt/blute_wound = affecting.get_wound_type(/datum/wound/blunt)
-				if(blute_wound && blute_wound.severity >= WOUND_SEVERITY_MODERATE)
-					//At this point we'll be doing the medical action that's not a grab maneuver
-					return
-				//Dislocation
-				. = TRUE
-				user.changeNext_move(4 SECONDS)
-				target.visible_message(span_danger("[user.name] twists [target.name]'s [affecting.name] violently!"), \
-						span_userdanger("[user.name] twists your [affecting.name] violently!"), ignored_mobs=user)
-				to_chat(user, span_danger("You start twisting [target.name]'s [affecting.name] violently!"))
-				if(do_after(user, 4 SECONDS, target))
-					target.visible_message(span_danger("[user.name] dislocates [target.name]'s [affecting.name]!"), \
-						span_userdanger("[user.name] dislocates your [affecting.name]!"), ignored_mobs=user)
-					to_chat(user, span_danger("You dislocate [target.name]'s [affecting.name]!"))
-					affecting.force_wound_upwards(/datum/wound/blunt/moderate)
-					log_combat(user, target, "dislocates", "the [affecting.name]")
+	switch(deprecise_zone(user.zone_selected))
+		if(BODY_ZONE_HEAD)
+			if(!(target.body_position == LYING_DOWN))
+				return
```

Comment:
I'd also recommend having a `to_chat` or bubble alert explaining why the headslam is failing.  
```suggestion
				return FALSE
```